### PR TITLE
Network Engineering Contribution

### PR DESCRIPTION
**Pull request by ZebGirouard**
@brianamarie and @atjonathan , pushing up my scripts, vid links, and slides link.  Let me know if I'm missing anything.

### CODE REVIEW

**Comment by atjonathan** [comment]
@ZebGirouard, please make a `video-list.json` file similar to https://github.com/ZebGirouard/hikes/blob/24be748574c4608792198b7ffb57fe2dd04567ce/video-categories/chrome-developer-tools/video-list.json.

Tagging @brianamarie and @QuincyLarson for feedback on the videos.

**Comment by ZebGirouard** [comment]
Cool @atjonathan , added the file in network-engineering directory.  Not sure what the threshold for "complete" is, so not sure if "true" is correct, but otherwise looks like it follows the pattern.

**Comment by atjonathan** [comment]
### **LGTM :+1:**
Just need reviewing of the videos :+1:

**Comment by atjonathan** [comment]
@brianamarie, update on this?

**Comment by brianamarie** [comment]
@QuincyLarson Do you see overlap here between the videos we worked on last month in SF? If so, how do you think we should move forward?

**Comment by QuincyLarson** [comment]
@ZebGirouard Thank you for your extreme patience. I enjoyed watching this. The content is great. Here is what I recommend we do:
1) take a look at the networking videos we already have. These really just scratch the surface, but there is some overlap between them and yours. Here's the full playlist: https://www.youtube.com/playlist?list=PLWKjhJtqVAblzbwhT83fRh5nNSHqywxrw
2) re-record the topics haven't been covered with a bigger white board and a better camera angle. See if you can draw the 7-layer model on the whiteboard instead of showing a tiny-font table, without using slides.

Your delivery and explanation were both excellent.

File: video-categories/network-engineering/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+##Network Engineering Hikes
```
Comment (atjonathan) [inline video-categories/network-engineering/README.md]: ~~`##Network Engineering Hikes`~~ `## Network Engineering Hikes`

**Comment by atjonathan** [comment]
@ZebGirouard, any update on this?

**Comment by ZebGirouard** [comment]
@atjonathan , sorry, but I haven't had time to work on this as other commitments have taken over my time.  I would really like to take time to resubmit these, but I don't think I can make any guarantees any time soon.

**Comment by atjonathan** [comment]
@ZebGirouard, I'll close as stale. Feel free to reopen when you get some free time 😃

**Pull request closed without merging**